PWM.py

```diff
@@ -0,0 +1,18 @@
+#code works along side GPIO.py
+#the init in GPIO.py must be called before any of these functions are called
+
+import RPi.GPIO as GPIO
+
+class signal:
+    def setPin(signal, pin, frequency):
+        GPIO.setup(pin, GPIO.OUT)
```

Comment:
didn't you wrap this function in [GPIO.py](https://github.com/wally-wally-wally/firmware/blob/main/GPIO.py#L7)? you should use that here instead.